usr/sbin/Makefile

```diff
@@ -23,85 +23,118 @@ ILIST = ${PROGRAMS} IPFL.README
 IPFL.README_IDIR = /usr/lpp/ipfl/
 IDIR = /usr/sbin/
 
```

Comment:
These changes will not invoke Makefile presents in "./64" directory.
Current makefile can create 64 bit binary for all utilities,but then we will not have binary with <binary_name> and <binary_name>64 ,based on the architecture(i.e. 32 bit and 64 bit).